features/step_definitions/search_steps.rb

```diff
@@ -15,7 +15,7 @@
 And /^the search results should be unique$/ do
   results = []
   page.all(".results-list li a").each_with_index do |item, idx|
-    results << item.text + page.all(".results-list li p")[idx].text
+    results << item[:href] + page.all(".results-list li p")[idx].text
```

Comment:
If we're comparing the URLs, do we still need to compare the descriptions? (I'm guessing that's what `page.all(".results-list li p")[idx].text` does.)